src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int maxNumber = 100;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+    private Ball ball = new Ball();
+
+    public Ball getRandomBall() {
```

Comment:
with the current implementation getRandomBall dont generate a new ball, just continuously update existing one 

so, Ball ball sholdn't be class-level field

Reply:
Understood and fixed. But I don't understand one moment - in this case I create 4 objects (One obj in class Lottery and 3 of them in Application) getRandomBall have only one obj just continuously update existing one and then transfer the result in another 3 objects (balls I created in Application class). But if ball shouldn't be class-level field then I create 6 objects instead of 4 and have the same result in final. So basically I do not understand why we are doing something like this. Thanks.

Reply:
no need to create a ball object in the Application because you have the #getRandomBall method for this. 

see comment above.